components/navigation/communityItems.js

```diff
@@ -1,21 +1,19 @@
-import IconTools from '../icons/Tools'
 import IconGithubOrganization from '../icons/GithubOrganization'
 import IconSlack from '../icons/Slack'
 import IconContributing from '../icons/Contributing'
 import IconDashboard from '../icons/Dashboard'
 import IconTSC from '../icons/TSC'
 import IconMeetings from '../icons/Meetings'
 import IconNewsroom from '../icons/Newsroom'
-
+import Ambassador from '../icons/Ambassador'
 
 export default [
-
-  { icon: IconTools, title: 'Tools & Services', href: '/tools', description: 'Explore the tools and services our awesome community has created.' },
   { icon: IconGithubOrganization, title: 'GitHub Organization', href: 'https://github.com/asyncapi', target: '_blank', description: 'Want to sneak in the code? Everything we do is open-sourced in our GitHub organization.' },
   { icon: IconSlack, title: 'Slack Workspace', href: 'https://asyncapi.com/slack-invite', target: '_blank', description: `Need help? Want to share something? Join our Slack workspace. We're nice people :)` },
   { icon: IconContributing, title: 'Contributing', href: 'https://github.com/asyncapi?type=source#-contribute-to-asyncapi', target: '_blank', description: `We are always welcoming and looking for contributions. If you are interested check out our contribution guide.` },
   { icon: IconTSC, title: 'Technical Steering Committee', href: '/community/tsc', description: 'Get to know what is a TSC member, how you can become one, and meet our current TSC members.' },
+     { icon: Ambassador, title: 'Ambassadors', href: '/community/ambassadors', description: 'Passionate about APIs? Become an AsyncAPI Ambassador and help shape the future of APIs.' },
   { icon: IconDashboard, title: 'Dashboard',href: '/community/dashboard', description: `Just need a good first issue to start your contribution journey? or want to see what topics are hot in discussion?`},
-  { icon: IconMeetings, title: 'Meetings', href: '/community/meetings', description: 'See what meetings are organized under AsyncAPI umbrella and join one of them.' },
+  { icon: IconMeetings, title: 'Meetings', href: '/community/events', description: 'See what meetings are organized under AsyncAPI umbrella and join one of them.' },
```

Comment:
Should the titile for this option be changed to something else for this page now, as we are not only restricted to Meetings but rather we are more focused on the events too like AsyncAPI Conference?
WDYT @derberg @alequetzalli

Reply:
It's best if we rename the title to "Events"